DopplerCustomDomain/CustomDomainProvider/IServiceNameResolver.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DopplerCustomDomain.CustomDomainProvider
+{
+    public interface IServiceNameResolver
+    {
+        Services Resolve();
```

Comment:
```suggestion
        string Resolve(string publicName);
```

It should receive the public name (from the client, in the request), ie. `forms-landing`, and it should return the Traefik's service name, ie `doppler_forms_service_prod@docker`.

Am I missing something?

Reply:
Here we added the implementation #4